docs/release_notes/2.2.1/migration_notes.md

```diff
@@ -0,0 +1,21 @@
+
+
+# Migration to 2.2.1 notes and tips
+
+
+### Using FOBS to serialize/deserialize data between Client and Server
+
+* prior to NVFLARE 2.1.4, NVFLARE is using python pickle to transfer data between client and server, since 2.1.4, 
+we switched to Flare Object Serializer (FOBS), you might experience failures if your code is still using Pickle. 
```

Comment:
It's not so much about using pickle but just trying to send data types that are not supported by FOBS.